src/Collections/Examples.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Swagger package.
+ *
+ * (c) EXSyst
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace EXSyst\OAS\Collections;
+
+use EXSyst\OAS\AbstractObject;
+use EXSyst\OAS\Example;
+use function EXSyst\OAS\referenceOr;
```

Comment:
I'm not a big fan of functions as they need to be loaded on every request, even if not needed. Can't they be in `AbstractObject` instead ?

Reply:
Oh that's too bad, I'm big fan of them, don't understand why people avoid them.
It is not big deal for me but:
 - Will that load be even distinguishable from 0? Every class is loaded using include, if your app is written on any framework you will have good hundred of includes just those framework classes, not talking about userland. Opcache and stuff
 - Purity, except this ᵗᶦᶰʸ include overhead there is no reason to put it as a static method, I mean it would stay the same function just with classname attached to its name.

Reply:
Ok let's keep it as is for now, it's not such a big deal indeed.